src/apolloClient.tsx

```diff
@@ -16,7 +16,8 @@ const errorLink = onError(({ graphQLErrors, networkError }) => {
       )
       switch (extensions.code) {
         case 'UNAUTHENTICATED':
-          window.location.href = '/login'
+          console.error('user not logged in')
+          //window.location.href = '/login'
```

Comment:
This line was causing us to redirect to `/login` twice, once here and once from the `useEffect` in the `authContext`. That was causing a race condition when adding the `redirectTo` flag. Changing this resolved that but I'm not sure if this creates some other issue.